drivers/FlashIAP.cpp

```diff
@@ -25,6 +25,7 @@
 #include <algorithm>
```

Comment:
don't change the file attributes (see `drivers/FlashIAP.cpp 100644 → 100755` above), also for other files